docker-compose.testcafe.yml

```diff
@@ -14,6 +14,7 @@ services:
     environment:
       XVFB_SCREEN_WIDTH: ${SCREEN_WIDTH-1280}
       XVFB_SCREEN_HEIGHT: ${SCREEN_HEIGHT-720}
+      DRUPAL_DEFAULT_ACCOUNT_NAME: ${DRUPAL_DEFAULT_ACCOUNT_NAME}
```

Comment:
maybe default this to `admin` as well? 

`DRUPAL_DEFAULT_ACCOUNT_NAME: ${DRUPAL_DEFAULT_ACCOUNT_NAME:admin}`